packages/gafl-webapp-service/src/pages/licence-details/licence-type/licence-type.njk

```diff
@@ -11,23 +12,58 @@
     }
 %}
 
+{% set troutAndCoarse2Rod %}
+    <p class="govuk-body-m">This lets you fish non-migratory trout and freshwater fish</p>
+    <p class="govuk-body-m">Rods are for the individual licence holder and must be attended at all times</p>
+{% endset -%}
+
+{% set troutAndCoarse3Rod %}
+    <p class="govuk-body-m">This lets you fish non-migratory trout and freshwater fish</p>
+    <p class="govuk-body-m">Rods are for the individual licence holder and must be attended at all times</p>
+{% endset -%}
+
+{% set salmonAndSeaTrout %}
+    {{ govukWarningText({
+      text: "You must submit an annual catch return",
+      iconFallbackText: "Warning",
+      classes: "govuk-!-margin-top-6"
+    }) }}
+    <p class="govuk-body-m">You must report you yearly fishing activity in England and Wales if you have a salmon and sea trout licence, even if you did not fish.</p>
+    <p class="govuk-body-m">More details can be found at <a target="_blank" href="https://www.gov.uk/catch-return">report a catch return</a>.</p>
+{% endset -%}
+
+{% set items = [
+        {
+            text: "Trout and coarse, up to 2 rods",
+            value: data.licenseTypes.troutAndCoarse2Rod,
+            checked: payload['licence-type'] === data.licenseTypes.troutAndCoarse2Rod,
+            conditional: { html: troutAndCoarse2Rod }
+        }
+    ]
+%}
+
+{% if not data.permission.licenceLength or data.permission.licenceLength === '12M' %}
+    {{ items.push({
+        text: "Trout and coarse, up to 3 rods",
+        value: data.licenseTypes.troutAndCoarse3Rod,
+        checked: payload['licence-type'] === data.licenseTypes.troutAndCoarse3Rod,
+        conditional: { html: troutAndCoarse3Rod }
+    }) }}
+{% endif %}
+
+{{ items.push({
+    text: "Salmon and sea trout",
+    value: "salmon-and-sea-trout",
+    hint: { text: "Includes trout and coarse, up to 3 rods" },
+    checked: payload['licence-type'] === data.licenseTypes.salmonAndSeaTrout,
+    conditional: { html: salmonAndSeaTrout }
+}) }}
```

Comment:
This looks like we get two new items pushed onto items if licence length is not defined or it's 12 months - is that what's intended?

Reply:
yes that is correct.